.github/workflows/chatbot-test.yml

```diff
@@ -0,0 +1,24 @@
+name: Chat Bot Test
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/**'
+      - 'docker/**'
+      - 'intel_extension_for_transformers/**'
```

Comment:
Which part of the file change under this package will impact chatbot?

Reply:
all subfiles are needed for chatbot